### ISSUE TITLE
webpack dll

### ISSUE THREAD

**Issue by bsr203** [Drathal/react-playground#7]
I found that, using dll reduced my build + watch time considerably. I adapted it from this https://github.com/mxstbr/react-boilerplate/pull/495

just FYI, and I may attempt a pr if I get around it.

**Comment by Drathal** [comment]
@bsr203 yeah thats on my todo list. it would be great if you could help out
